lang/java/reef-tang/tang-test-jarA/src/main/java/org/apache/reef/tang/examples/A.java

```diff
@@ -19,5 +19,4 @@
 package org.apache.reef.tang.examples;
 
 public class A {
-  private int iAmAClass = 0;
```

Comment:
Ur, I agree with you on that there is not sufficient usage for that. 

I searched the historic usage of this and found that the field `iAmAClass` had been *public* since 0.1 (MS) ~ 0.12 (Apache REEF) before the following commit (it's yours.)

[REEF-441] Fix violations of VisibilityModifier checkstyle
https://github.com/apache/reef/commit/c9e1bf9b3d73d04e34e76236a6818477ed4f74e6

I guess it was changed into `private` due the the lack of usage at that time. The same reason of this PR. IMHO, this situation is not only for this field. The followings has the same situation.
* `TestClassLoaders.java`: `@Test` is commented out here.
* `tang-test-jarA`, `tang-test-jarB`, `tang-test-jarAB` projects: 

Should we remove all of them? I prefer to uncomment and revive them to enhance the test coverage of Apache REEF projects and also add some usage for the `public int iAmAClass`. How do you guys think about this, @tcNickolas , @markusweimer , @bgchun ?